common/lib/protocol-definitions/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/protocol-definitions",
-	"version": "1.2.0",
+	"version": "2.0.0",
```

Comment:
@tylerbutler FYI - I think it's still correct to bump in this PR, and once @ghoshkaj 's changes are in too then they can both go out with 2.0.0.